packages/sources/svelte/src/components/indicators/VtmnTag/test/VtmnTag.spec.js

```diff
@@ -88,5 +88,13 @@ describe('VtmnTag', () => {
       const { getByText } = render(VtmnTag, { href: 'http://example.com' });
       expect(getByText('Unit-test tag')).toBeVisible();
     });
+    test('Should apply size', () => {
```

Comment:
Add a test by default the component should have `vtmn-tag_size--medium`

Reply:
done